artifacts/definitions/Linux/Events/ProcessExecutions.yaml

```diff
@@ -17,20 +17,21 @@ required_permissions:
   - EXECVE
 
 parameters:
-  - name: pathToAuditctl
-    default: /sbin/auditctl
-    description: We depend on auditctl to install the correct process execution rules.
+  - name: pathToAugenrules 
+    default: /sbin/augenrules
+    description: We depend on augenrules to persist the correct process execution rules.
 
 sources:
   - query: |
-     // Remove default filter if any
-     LET _ <= SELECT * FROM execve(argv=[pathToAuditctl, "-d", "task,never"])     
      // Install the auditd rule if possible.
-     LET _ <= SELECT * FROM execve(argv=[pathToAuditctl, "-a",
-          "exit,always", "-F", "arch=b64", "-S", "execve", "-k", "procmon"])
+     // Remove default filter if any
+     LET _ <= SELECT * FROM execve(argv=["/bin/bash", "-c", "if [ ! -f /etc/audit/rules.d/vrr_clear_default.rules ]; then echo '-d task,never -i' > /etc/audit/rules.d/vrr_clear_default.rules; chmod 0640 /etc/audit/rules.d/vrr_clear_default.rules; " + pathToAugenrules + " --load; fi"])
+
+     // add rule to collect new process 
+     LET _ <= SELECT * FROM execve(argv=["/bin/bash", "-c", "if [ ! -f /etc/audit/rules.d/vrr_procmon_b64.rules ]; then echo '-a always,exit -F arch=b64 -S execve -k vrr_procmon' > /etc/audit/rules.d/vrr_procmon_b64.rules; chmod 0640 /etc/audit/rules.d/vrr_procmon_b64.rules; " + pathToAugenrules + " --load; fi"])
+
+     LET _ <= SELECT * FROM execve(argv=["/bin/bash", "-c", "if [ ! -f /etc/audit/rules.d/vrr_procmon_b32.rules ]; then echo '-a always,exit -F arch=b32 -S execve -k vrr_procmon' > /etc/audit/rules.d/vrr_procmon_b32.rules; chmod 0640 /etc/audit/rules.d/vrr_procmon_b32.rules; " + pathToAugenrules + " --load; fi"])
```

Comment:
maybe have both these rules in a single file, vrr_procmon.rules, as they are related?